src/commands/mod.rs

```diff
@@ -1416,24 +1416,28 @@ fn log_template(settings: &UserSettings) -> String {
     // TODO: If/when this logic is relevant in the `lib` crate, make this into
     // and enum similar to `ColorChoice`.
     let prefix_format = match settings.unique_prefixes().as_str() {
-        "brackets" => "shortest_prefix_and_brackets()",
-        "styled" => "shortest_styled_prefix()",
-        _ => "short()",
+        "brackets" => "shortest_prefix_and_brackets",
+        "styled" => "shortest_styled_prefix",
+        _ => "short",
     };
+    let desired_id_len = settings
+        .log_id_preferred_length()
+        .map(|l| l.to_string())
+        .unwrap_or_default();
```

Comment:
Nit: for readability, it's probably better to build `"{prefix_format}({desired_id_len})"` fragment here.

Reply:
Sounds good.